### PR TITLE
Small clean up and take backup of output before resuming

### PR DESCRIPTION
**Pull request by wpoely86**
This script should have a major overhaul but it's probably better to first focus #2
 
@boegel @JensTimmerman  please review?

### CODE REVIEW

File: base.sh [hunk 1]
```diff
@@ -39,29 +39,30 @@ job_pids_cache=""
...
 
 # make sure qsub is available
-module load jobs
+module load scripts
 
 jobname=${%(CSUB_JOBNAME)s}
 ## jobname_stripped is set in BASEHEADER
 scriptname="${jobname_stripped}.sh"
-localdir=$%(CSUB_SCRATCH_NODE)s/$jobname
+localdir=${%(CSUB_SCRATCH_NODE)s}/$jobname
```
Comment (JensTimmerman) [inline base.sh]: better ad quotes around this too?

File: base.sh [hunk 2]
```diff
@@ -446,13 +447,14 @@ endofjob () {
...
-       cp -a "$chkpoststage" /tmp/poststage
+       tmpdir=$(mktemp -d)
+       cp -a "$chkpoststage" $tmpdir/poststage
        chmod +x /tmp/poststage
        if (( %(cleanup_chkpt)d ))
        then
        		rm -Rf "$chkdir"
        fi
-       /tmp/poststage
+       $tmpdir/poststage
```
Comment (JensTimmerman) [inline base.sh]: quotes around $tmpdir ?

File: csub.py [hunk 3]
```diff
@@ -760,22 +753,21 @@ def parsetime(time_str):
...
         if base:
             # make sure it also works correctly for array jobs
             if arrayid:
                 arrayspec = arrayid
 
+            try:
+                f = open(base, "r")
+                basetxt = f.read()
+                f.close()
+            except Exception, err:
```
Comment (JensTimmerman) [inline csub.py]: dont' catch Exception, but the correct thing you're expecting, like OSError or IOError

Comment (JensTimmerman) [inline csub.py]: use `Exception as err` as not to have confusion with the case where you're trying to except two Exceptions (between brackets)

File: csub.py [hunk 4]
```diff
@@ -760,22 +753,21 @@ def parsetime(time_str):
...
+                print "Failed to read base script %s: %s" % (base, err)
```
Comment (JensTimmerman) [inline csub.py]: better exit after this exception? Or how can we proceed gracefully?

File: csub.py [hunk 5]
```diff
@@ -760,22 +753,21 @@ def parsetime(time_str):
...
-
-                try:
-                    f = open(base, "r")
-                    basetxt = f.read()
-                    f.close()
-                except Exception, err:
-                    print "Failed to read base script %s when adjusting job_time/chkpt_time: %s" % (base, err)
-                    sys.exit(1)
-
                 walltime_script = get_wall_time(basetxt)
```
Comment (JensTimmerman) [inline csub.py]: Won't this fail if you failed to read basetxt earlier and it is None ?

**Comment by wpoely86** [comment]
Remarks fixed, please rereview.

**Pull request merged**